--- ./zha-2.0.0/zha/application/const.py
+++ /srv/homeassistant/lib/python3.14/site-packages/zha/application/const.py
@@ -12,6 +12,7 @@
 import zigpy_zigate.zigbee.application
 import zigpy_ziggurat.zigbee.application
 import zigpy_znp.zigbee.application
+import zigpy_blz.zigbee.application
 
 ATTR_ACTIVE_COORDINATOR = "active_coordinator"
 ATTR_ARGS = "args"
@@ -123,6 +124,10 @@
         "XBee = Digi XBee Zigbee radios: Digi XBee Series 2, 2C, 3",
         zigpy_xbee.zigbee.application.ControllerApplication,
     )
+    blz = (
+        "BLZ = Bouffalo Lab Zigbee radios: BL702/4/6",
+        zigpy_blz.zigbee.application.ControllerApplication,
+    )
 
     @classmethod
     def list(cls) -> list[str]:
