--- zigpy-cli/zigpy_cli/const.py	2025-07-01 07:53:48.909632575 +0000
+++ /srv/homeassistant/lib/python3.13/site-packages/zigpy_cli/const.py	2025-07-01 07:07:04.411923023 +0000
@@ -14,6 +14,7 @@
     "zboss": "zigpy_zboss",
     "zigate": "zigpy_zigate",
     "znp": "zigpy_znp",
+    "blz": "zigpy_blz",
 }
 
 
@@ -58,6 +59,16 @@
             "zigpy_zboss.api": logging.DEBUG,
         },
     ],
+    "blz": [
+        {
+            "zigpy_blz.zigbee.application": logging.INFO,
+            "zigpy_blz.api": logging.INFO,
+        },
+        {
+            "zigpy_blz.zigbee.application": logging.DEBUG,
+            "zigpy_blz.api": logging.DEBUG,
+        },
+    ],    
     "zigate": [
         {
             "zigpy_zigate": logging.INFO,
@@ -76,7 +87,7 @@
         {
             "zigpy_znp": TRACE,
         },
-    ],
+    ]
 }
 
 RADIO_TO_PYPI = {name: mod.replace("_", "-") for name, mod in RADIO_TO_PACKAGE.items()}
